package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "pg": "^6.1.3",
-    "sequelize": "^3.30.2"
+    "sequelize": "3.30.2 - 4.2"
```

Comment:
It *is* odd. See here for some background: https://github.com/cockroachdb/sequelize-cockroachdb/pull/1#discussion_r105714302

I'd vote for listing it only as a peer dep, but I know @cuongdo was annoyed that that required a separate `npm install sequelize` step.